src/LanguageServer/Impl/Completion/TopLevelCompletion.cs

```diff
@@ -80,6 +81,14 @@ internal static class TopLevelCompletion {
             return new CompletionResult(items, applicableSpan);
         }
 
+        private static IReadOnlyList<IVariable> CollectVisibleVariables(IScope currentScope) {
+            var variables = new List<IVariable>();
+            foreach (var s in currentScope.EnumerateTowardsGlobal) {
+                variables.AddRange(s.Variables.Where(v => v.Source != VariableSource.ClassMember || s == currentScope));
```

Comment:
Is there a check like this in the analysis? Filters in the completion code tend to hide behaviors of the analysis (like allowing access to something that doesn't exist), but if there's equivalent code there then it's probably not a big deal...